src/definitions/modules/dropdown.less

```diff
@@ -572,6 +572,17 @@ select.ui.dropdown {
     display: none;
   }
 
+  /* User addition item */
+  @supports (position: sticky) {
```

Comment:
I thought about this again: We should remove the @supports declaration again and instead add `position:-webkit-sticky `in addition to `position: sticky` . Why? To [support Safari versions up to 12](https://caniuse.com/?search=sticky) which is from 2018 and most probably still widely used (at least more than ie11), so resulting in

I also would prefer `stuck` instead of sticky to make sure it does not interfere with the sticky component and because the addition is definately stuck at the top of the menu, not just sticky (which basically means is dynamic and not always stuck)

I hope you are ok with that (?) 🙂 

```css
    .ui.dropdown > .menu > .stuck.addition.item:first-child {
      position: -webkit-sticky;
      position: sticky;
      top: 0;
      box-shadow: @selectionVisibleBoxShadow;
      background: @menuBackground;
      z-index: @additionZIndex;
    }
  ```
We did it the same way for the new `stuck table` variant and added a hint in the docs that that feature needs a modern browser.

Reply:
Yeah I'm completely OK, this is for sure more interesting to support Safari rather than ie11!
Modified to `stuck` too, understandable.

I'll add another hint in the doc as you made for the table this afternoon :)